### PR TITLE
Added rule to prevent too early booking

### PR DESCRIPTION
**Pull request by x183**
Added a temporary workaround for issue #26.

### CODE REVIEW

**Comment by molleer** [comment]
It works, but please use the `is_admin` variable instead of looking at `event.booked_as` to determine weather or not the user is admin. You can see how to use it here: https://github.com/cthit/bookIT-node/blob/0bba4c7a5c51abfbbd5ea2404ccd511ddb0e084f/backend/src/services/event.service.ts#L228

Also, you should be able of creating a new branch in the cthit/bookIT-node repo instead of a your forked repo since you are now digIT

**Comment by x183** [comment]
> It works, but please use the `is_admin` variable instead of looking at `event.booked_as` to determine weather or not the user is admin. You can see how to use it here:
> 
> https://github.com/cthit/bookIT-node/blob/0bba4c7a5c51abfbbd5ea2404ccd511ddb0e084f/backend/src/services/event.service.ts#L228
> 
> Also, you should be able of creating a new branch in the cthit/bookIT-node repo instead of a your forked repo since you are now digIT

Is fixed 😜

File: backend/src/services/event.service.ts [hunk 1]
```diff
@@ -23,6 +24,13 @@ const validEvent = async (
     };
   }
 
+  if ((new Date(event.start) > new Date(Date.now() + 5443200000)) && is_admin) {
```
Comment (molleer) [inline backend/src/services/event.service.ts:27]: Shouldn't it be `&& !is_admin`?
Now the 9 week rule only applies to admins..

**Review by molleer — CHANGES_REQUESTED** [review]

**Review by molleer — APPROVED** [review]
Look nice!

**Pull request merged**